invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/ShareOptions/AccessRequests/AccessRequestsTab.js

```diff
@@ -80,6 +101,9 @@ export class AccessRequestsTab extends Component {
             allow_guest_requests: record.parent.access.settings.allow_guest_requests,
             accept_conditions_text:
               record.parent.access.settings.accept_conditions_text,
+            secret_link_expiration: this.translateExpirationDate(
```

Comment:
What about simply doing:
```suggestion
            secret_link_expiration: record.parent.access.settings.secret_link_expiration || 0;
```